lib/screens/admin_order_detail_page.dart

```diff
@@ -0,0 +1,585 @@
+          fontWeight: FontWeight.w600,
+        )
+      ),
+      centerTitle: true,
+      backgroundColor: paletteLightGreyColor,
+      elevation: 0.0,
+      iconTheme: IconThemeData(color: paletteBlackColor),
+    );
+  }
+
+  Widget _buildBody(BuildContext context) {
+    return Container(
+      color: paletteLightGreyColor,
+      child: ListView(
+        children: <Widget>[
+          _buildOrderPlacedDateText(),
+          _buildOrderStatusDescriptionBar(),
+          _buildOrderItemList(),
+          _buildOrderPriceDescriptionBar(),
+          SizedBox(height: 60.0),
+          _buildBuyerDetails(),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildOrderPlacedDateText() {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.end,
+      children: <Widget>[
+        Container(
+          padding: EdgeInsets.only( right: 16.0 ),
+          child: Text(
+            DateFormat('yyyy.MM.dd').format(widget.order.placedDate) + '受付',
+            style: TextStyle(
+              color: paletteBlackColor,
+              fontSize: 12.0,
+              fontWeight: FontWeight.normal,
+            ),
+          ),
+        ),
+      ],
+    );
+  }
+
+  Widget _buildOrderStatusDescriptionBar() {
+    return Container(
+      padding: EdgeInsets.symmetric( vertical: 8.0 ),
+      child: Container(
+        height: 56,
+        color: widget.order.status == 
+            OrderStatus.pending ? paletteDarkRedColor: paletteDarkGreyColor,
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            Text(
+              widget.order.getOrderStatusText(),
+              style: TextStyle(
+                color: kPaletteWhite,
+                fontSize: 16.0,
+                fontWeight: FontWeight.w600,
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Widget _buildOrderItemList() {
+    return Container(
+      padding: EdgeInsets.symmetric(horizontal: 16.0),
+      child: StreamBuilder<QuerySnapshot>(
+        stream: Firestore.instance
+            .collection(constants.DBCollections.products)
+            .snapshots(),
+        builder: (context, snapshot) {
+          if (!snapshot.hasData) {
+            return LinearProgressIndicator();
+          }
+
+          return Column(
+            children: widget.order.items.map( (orderItem) {
+              DocumentSnapshot productSnapshot = snapshot.data.documents.firstWhere(
+                (snapshot) => snapshot.documentID == orderItem.productId
+              );
+              return _buildOrderItemListRow(
+                orderItem, Product.fromSnapshot(productSnapshot)
+              );
+            }).toList(),
+          );
+        }
+      ),
+    ); 
+  }
+
+  Widget _buildOrderItemListRow(OrderItem orderItem, Product product) {
+    return Container(
+      decoration: BoxDecoration(
+        border: Border(
+          bottom: BorderSide(color: const Color(0xFFA2A2A2)),
```

Comment:
I see that this color has been used in multiple places (outside your PR as well). Can you please add a const for this in the colors.dart instead of hardcoding it here?